MV.Wildebeest.Core/source/test/etc/InvalidXml/InvalidSampleResourcesUsesAssertionGroup.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <resource type="co.mv.wb.MySqlDatabase" id="0d39b8fb-5b5c-48cd-845c-9c4d55f94303" name="Product Catalogue Database">
     <assertionGroups>
-        <assertionGroup name="group1">
+        <assertionGroup name="group1" id="af32a1e6-ae4c-42b8-a1fd-b11b461f8e39">
```

Comment:
Is ID a required field?  It probably should be.

Reply:
I figured that all parts of XML have name and id so I just added the id in assertionGroup just to make it uniform.    I think it will come in handy In the future, when you have a visual editor for the resource files.  Should I retain it?

Reply:
Yes we should retain it :)  You're spot on with your decision.  My question was ambiguous - I'm asking if it is enforced by the XSD to have an ID.  If not then we should backlog to make that so.